typescript/packages/plugins/solana-nfts/package.json

```diff
@@ -14,11 +14,13 @@
     "dependencies": {
         "@goat-sdk/core": "workspace:*",
         "@metaplex-foundation/digital-asset-standard-api": "1.0.0",
-        "@metaplex-foundation/mpl-bubblegum": "3.1.0",
+        "@metaplex-foundation/mpl-bubblegum": "4.2.1",
         "@metaplex-foundation/umi": "^0.9.2",
         "@metaplex-foundation/umi-bundle-defaults": "0.9.2",
         "@metaplex-foundation/umi-web3js-adapters": "0.8.10",
         "@solana/web3.js": "catalog:",
+        "got": "5.6.0",
```

Comment:
intentional?

Reply:
Don't get me started... without manually installing this dependency it tries to install a version that requires node <7 and fails... cost me 4 hours a few days ago